src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketDelete.java

```diff
@@ -103,11 +108,15 @@ public void run(ActionContext context) throws Exception {
           String.format("Unable to access or create bucket %s. ", gcsPath.getBucket())
             + "Ensure you entered the correct bucket path and have permissions for it.", e);
       }
-      gcsPaths.add(new Path(gcsPath.getUri()));
+      if (gcsPath.getUri().toString().contains("*")) {
```

Comment:
I'm wondering whether we can handle it uniformly for wild and non-wild case ? non-wild case is just a special case of wild case , right ?

Reply:
For non-wild case, we do not need to iterate Blobs (storage path list) for each gcspath. If we use the same logic for wild and non-wild case, I am worries if time for searching the filed to delete will increase?